wiki/Interface/fr.md

```diff
@@ -48,32 +46,32 @@ Accédez à cet écran en cliquant sur votre profil se trouvant en haut à gauch
 - Cliquez sur `Multi` (`M`) pour jouer avec d'autres personnes.
 - Cliquez sur `Back` pour revenir au menu principal.
 
-## Le hall multijoueur
+## Lobby multijoueur
 
-*Pour plus d'informations: [Multi](/wiki/Multi)*
+*Page principale : [Multi](/wiki/Multi)*
 
-![](img/multi-lobby-FR.jpg "Ecran de sélection de salle")
+![](img/multi-lobby-FR.jpg "Écran de sélection de salle")
 
 ![](img/multi-room-FR.png "Salle multijoueur")
 
-- \[1\] Votre rang dans la salle. Vous pouvez être soit l'hôte (si vous avez crée la salle ou si l'hôte précédent vous a donné ce rôle), soit un participant. Ce rang est également affiché à côté de votre nom.
-- \[2\] Votre profil.
-- \[3\] Le jukebox.
-- \[4\] La liste des joueurs. On peut y voir le nom des joueurs, leur rang (hôte ou participant), les [mods](/wiki/Game_modifier) qu'ils ont choisis (s'il y en a, voir #7), leur classement sur osu!, ainsi que leur équipe (si le type de partie est réglé sur `Team VS` ou `Tag-team VS`).
-- \[5\] Le nom de la salle et les paramètres du mot de passe.
-- \[6\] La beatmap choisie. Elle est montrée comme dans l'écran de sélection de beatmaps dans le mode Solo.
-- \[7\] Les [mods](/wiki/Game_modifier) que vous avez activé (voir #12), ainsi que l'option pour les sélectionner. L'option intitulée "Mods libres" décide de si les joueurs peuvent choisir leurs propres mods ou non. Si elle est activée, les joueurs peuvent choisir n'importe quelle combinaison de mods *sauf les mods altérant la vitesse de la beatmap, c'est-à-dire [Double Time](/wiki/Game_modifier/Double_Time) et [Half Time](/wiki/Game_modifier/Half_Time)*. Si elle est désactivée, l'hôte décide de quels mods seront utilisés. L'hôte peut choisir les mods altérant la vitesse de la beatmap peu importe si l'option "Mods libres" est activée ou non.
-- \[8\] Le type de partie et les conditions de victoire.
-- \[9\] Le bouton "Je suis prêt !".
-- \[10\] La [console de tchat](/wiki/Chat_Console).
-- \[11\] Le bouton "Quitter la salle".
-- \[12\] Là où se trouvent les mods activés par l'hôte.
+1. Votre rang dans la salle. Vous pouvez être soit l'hôte (si vous avez créé la salle ou si l'hôte précédent vous a donné ce rôle), soit un participant. Ce rang est également affiché à côté de votre nom.
```

Comment:
it seems like the english article is inconsistent with list style. looks like a simple fix for all translations, so it can be left another pr

Reply:
has an issue already https://github.com/ppy/osu-wiki/issues/867